CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.10.18
+
+<!-- generated comparing v5.10.17..master -->
+
+_Dec 12, 2022_
+
+A big thanks to the 18 contributors who made this release possible.
+This release contains various 🐛 bug fixes, 📚 documentation, and ⚙️ infrastructure improvements.
+
+### `@mui/material@5.10.18`
+
+- [Alert] Update icon color in all variants (#35414) @danilo-leal
+- [Select] Fix `MenuProps.PopoverClasses` being overriden (#35394) @vitorfrs-dev
+- [SwipeableDrawer] fixed typescript warning "prop open undefined" (#34710) @kraftware
+
+### `@mui/icons-material@5.10.18`
+
+- [icons] Restore the PhoneInTalk icons (#35409) @michaldudak
+
+### `@mui/system@5.10.18`
+
+- [system] Enable configuring the `sx` prop in the `theme` (#35150) @mnajdova
```

Comment:
This deserves a highlight, it fixes at least three opened issues :)

Reply:
I would even suggest changing the minor to 5.11.0 because of this change. It affects not only the system, but the other packages too (`@mui/material` and `@mui/joy`).

Reply:
All right, minor it is!

Reply:
Final comment, let's add the experimental breaking change here :) the rest looks good

Reply:
It's in the PR description: https://github.com/mui/material-ui/pull/35150